samples/resource-server-sample/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
```

Comment:
Please modify the build to follow the [helloworld](https://github.com/spring-projects-experimental/spring-authorization-server/blob/master/samples/helloworld/spring-authorization-server-samples-helloworld.gradle) sample, which uses the [spring-build-conventions](https://github.com/spring-gradle-plugins/spring-build-conventions) plugin (same as Spring Security).

The build file should be renamed to `spring-authorization-server-samples-oauth2resourceserver.gradle`

Reply:
Done.